src/app/_services/sitting-records-service/sitting-records.service.ts

```diff
@@ -0,0 +1,95 @@
+import { HttpClient } from "@angular/common/http";
+import { Injectable } from "@angular/core";
+import { SittingRecordsPostBody } from "../../_models/addSittingRecords.model";
+import { CookieService } from 'ngx-cookie-service';
+import { DateService } from "../date-service/date-service";
+
+@Injectable({
+    providedIn: 'root',
+})
+
+export class SittingRecordsService{
+    constructor(
+        private dateSvc: DateService,
+        private http: HttpClient,
+        private readonly cookies: CookieService
+    ){}
+
+    postNewSittingRecord(newSittingRecords: SittingRecordsPostBody) {
+        const headers = {
+            'Content-Type': 'application/json',
+        };
+
+        return this.http.post('/sittingrecord/add', { sittingRecords: newSittingRecords }, { headers: headers});
+    }
+
+
+    createNewSRPostObj(joh: any, tribunalService: any, dateSelected: any, venue: any, period:any) {
+        console.log(joh.johRole)
```

Comment:
Can anything be done about these warnings?

Reply:
Updated types of all params